checkstyle-suppressions.xml

```diff
@@ -10,4 +10,5 @@
     <suppress checks=".+"
               files="InjectedTest.java"/>
     <suppress checks="MagicNumber" files="BuildLogIndication.java"/>
+    <suppress checks="HiddenField" files="PluginImpl.java"/>
```

Comment:
instead of hiding bad code you should try to fix them, If you have a legitimate reason to ignore an error it should be stated in the code with a `//CS IGNORE...` statement.